PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditRequest.kt

```diff
@@ -7,11 +7,11 @@ import com.braintreepayments.api.core.ExperimentalBetaApi
  *
  * Note: **This feature is in beta. It's public API may change in future releases.**
  *
- * @property editPaypalVaultId PayPal vault ID to edit
- * @property merchantAccountId optional ID of the merchant account; if one is not provided the default will be used
+ * @property editPayPalVaultId PayPal vault ID to edit
  */
 @ExperimentalBetaApi
-data class PayPalVaultEditRequest(
-    val editPaypalVaultId: String,
-    val merchantAccountId: String? = null,
-)
+open class PayPalVaultEditRequest(
```

Comment:
This should have a JvmOverloads constructor (see [here](https://github.com/braintree/braintree_android/blob/main/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.kt#L29))